plenum/common/util.py

```diff
@@ -554,3 +554,24 @@ def min_3PC_key(keys) -> Tuple[int, int]:
 
 def max_3PC_key(keys) -> Tuple[int, int]:
     return max(keys, key=lambda k: (k[0], k[1]))
+
+
+from sortedcontainers import SortedDict as _SortedDict
+if 'peekitem' in dir(_SortedDict):
+    SortedDict = _SortedDict
+else:
+    # Since older versions of `SortedDict` lack `peekitem`
```

Comment:
Why don't depend on a particular version of sortedcontainers (using == in setup.py) having all necessary functionality?